src/components/bookForm.jsx

```diff
@@ -33,17 +33,17 @@ function BookForm() {
   return (
     <form>
       <h2>ADD NEW BOOK</h2>
```

Comment:
- [ ] This title is not strictly black in color but gray, kindly use the right color

zeplin design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/28562c90-9a20-409a-b62e-fc3c64a71f59)

your design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/f0cf3697-f77c-4298-880d-ccb970952a01)